HTMLSamples/data-chart/chart-title.html

```diff
@@ -26,7 +26,11 @@
                 height: "400px",
                 dataSource: lastFiveYears,
                 title: "$$(Chart_title_energy_production)",
+                titleTextColor: "#2e9ca6",
+                titleTextStyle: "20pt Verdona",
```

Comment:
Can this be something sans-serif to match the chart overall styling? Like `"20pt Arial"`? Same for the `subtitleTextStyle`